ui/src/app/utils/runtime-utils.tsx

```diff
@@ -1243,6 +1244,14 @@ export const withRuntimeStore = () => (WrappedComponent) => {
   };
 };
 
+export const withUserAppsStore = () => (WrappedComponent) => {
```

Comment:
Followed pattern of `withRuntimeStore` to make a high order component to pass the userAppsStore. Interactive-notebook is currently a class based component, so it cannot make use of hooks. 

It might be useful, in a future PR, to convert Interactive-notebook from a class based component to a functional component.

Reply:
Additionally, class components are now considered [legacy](https://react.dev/reference/react/Component#:~:text=API%20REFERENCE-,LEGACY%20REACT%20APIS,-Component).

Reply:
What code is ensuring that the store is set?

Reply:
Thanks Peter. Definitely onboard with switching this to be a functional component, but it seemed out of scope for this particular PR. Hoping we can do that soon.